src/links/HttpLink.ts

```diff
@@ -0,0 +1,35 @@
+import { createApolloFetch } from 'apollo-fetch';
+import * as Observerable from 'zen-observable';
+
+import Link from '../';
+
+export default (uri: string) => {
+  // create HTTP utility
+  const fetcher = createApolloFetch({ uri });
+
+  // create Link from request and previous link
+  return new Link((request, prev = Link.empty()) => {
+    // hook up Observerable
+    return new Observerable(observer => {
```

Comment:
Really like this way of constructing Observables, I think we should use it in our own implementation of Observable or just use zen-observable